src/pudl/metadata/fields.py

```diff
@@ -2309,19 +2796,44 @@
     },
     "respondent_type": {
         "type": "string",
+        "description": (
+            "Whether a respondent to the FERC form 714 is a utility or a balancing "
+            "authority."
+        ),
```

Comment:
This is pretty specific to 714, maybe we should add a `_714` suffix to the column?